tests/installation/partitioning_full_lvm.pm

```diff
@@ -16,38 +16,55 @@ use warnings;
 use base 'y2logsstep';
 use testapi;
 use partition_setup;
+use version_utils 'is_storage_ng';
 
 sub run {
-    wipe_existing_partitions;
-    if (check_var('ARCH', 's390x')) {    # s390x need /boot/zipl on ext partition
+    create_new_partition_table;
+    if (check_var('ARCH', 's390x') || get_var('UNENCRYPTED_BOOT')) {    # s390x need /boot/zipl on ext partition
         addpart(role => 'OS', size => 500, format => 'ext2', mount => '/boot');
     }
-    elsif (check_var('ARCH', 'ppc64le')) {    # ppc64le need PReP /boot
+    elsif (check_var('ARCH', 'ppc64le')) {                              # ppc64le need PReP /boot
         addpart(role => 'raw', size => 500, fsid => 'PReP');
     }
+    elsif (is_storage_ng) {
+        # Storage-ng has GPT by defaut, so need bios-boot partition
+        addpart(role => 'raw', fsid => 'bios-boot', size => 1);
+    }
     addpart(role => 'raw', encrypt => 1);
     assert_screen 'expert-partitioner';
-    send_key 'alt-s';                         # select System view
-    for (1 .. 2) {
-        send_key 'down';                      # select Volume Management
-    }
-    send_key $cmd{addpart};                   # add
-    send_key 'down';
+    send_key 'alt-s';                                                   # select System view
+    send_key_until_needlematch('volume_management_feature', 'down');    # select Volume Management
+    send_key $cmd{addpart};                                             # add
     wait_still_screen 2;
     save_screenshot;
-    send_key 'ret';                           # create volume group
+    send_key 'down';
+    send_key 'ret';                                                     # create volume group
     assert_screen 'partition-add-volume-group';
+    send_key 'alt-v';                                                   # volume group name
     type_string 'vg-system';
-    send_key 'alt-d';                         # add all
+    send_key 'alt-d';                                                   # add all
     wait_still_screen 2;
     save_screenshot;
-    send_key $cmd{finish};
+    send_key(is_storage_ng() ? $cmd{next} : $cmd{finish});
     addlv(name => 'lv-swap', role => 'swap', size => 2000);
     assert_screen 'expert-partitioner';
-    addlv(name => 'lv-root', role => 'OS');
+
+    if (is_storage_ng) {
+        # Mount point is not preselected for OS role in storage-ng
+        record_soft_failure("bsc#1073854");
+        addlv(name => 'lv-root', role => 'OS', mount => '/');
+    }
+    else {
+        addlv(name => 'lv-root', role => 'OS');
+    }
     assert_screen 'expert-partitioner';
     send_key $cmd{accept};
-    assert_screen 'partitioning-full-lvm-encrypt';
+    if (get_var('UNENCRYPTED_BOOT')) {
+        assert_screen 'partitioning-full-lvm-encrypt-unencrypted-boot';
+    }
+    else {
+        assert_screen 'partitioning-full-lvm-encrypt';
```

Comment:
I think this needle is wrong. Because of the bug that the lvm is actually not encrypted, it is not reflected on this overview. This needle should be a workaround needle, pointing to an existing bsc.

Reply:
I'll come to you to discuss this one

Reply:
As spoken, I mean that:

- http://g226.suse.de/tests/554#step/partitioning_full_lvm/55
- http://g226.suse.de/tests/566#step/partitioning_full_lvm/53

Are matching an overview without encrypted lvm like shown here:

- http://g226.suse.de/tests/552#step/partitioning_full_lvm/47

----

Actually it is only about adding the property **workaround** to those needles.

Reply:
well, and creating a bug for that.

And changing the name of the files, to include bsc#x